websites/R/Rádio Animu/presence.ts

```diff
@@ -29,6 +29,8 @@ presence.on("UpdateData", async () => {
 	};
 
 	if (playing) {
+		presenceData.type = ActivityType.Listening;
+		presenceData.name = "Rádio Animu";
```

Comment:
not really necessary?

Reply:
True, but it is a radio station, it makes sense to display it as a listening activity, you don't _play_ a station, you **listen** to it.

The name is there to avoid the situation in this screenshot, where the name isnt displayed in the popup.

![Screenshot_20240925_222121](https://github.com/user-attachments/assets/8dd604b2-68c6-4440-9be6-c1b076274e4d)

Reply:
Could you try this without the .name in the latest alpha version.
If you don't have access to it @ me in discord.

The .name should be defaulted to the presence name by the extension, so then the code is not needed.

Reply:
Makes sense, i will try it out on the alpha version when get the access to it.